dev/AppNotifications/AppNotificationManager.cpp

```diff
@@ -167,6 +168,28 @@ namespace winrt::Microsoft::Windows::AppNotifications::implementation
         }
     }
 
+    void AppNotificationManager::Register()
+    {
+        RegisterHelper(winrt::hstring{} /* displayName */, std::wstring{} /* iconFilePath */);
+    }
+
+    void AppNotificationManager::Register(hstring const& displayName, winrt::Uri const& iconUri)
+    {
+        THROW_HR_IF_MSG(E_ILLEGAL_METHOD_CALL, AppModel::Identity::IsPackagedProcess(), "Not applicable for packaged applications");
+
+        THROW_HR_IF(E_INVALIDARG, (displayName != winrt::hstring{}) xor (iconUri != nullptr));
+
+        std::wstring iconFilePath{};
+        if (iconUri != nullptr)
```

Comment:
Remove this check please.  Its probably better to crash if this is null below.

Reply:
It isn't probably need after all. I thought we wanted to have both valid DisplayName and IconUri to complete the asset processing thing.